src/styles/02-molecules/input/input.scss

```diff
@@ -0,0 +1,26 @@
+.input {
+  
+  &__label {
+    @include space-inline-xs;
```

Comment:
Label is currently a wrapper around the actual `<label>` element and the required message, so this `space-inline-xs` isn't getting applied where you want it to:

![Screen Shot 2020-04-10 at 4 41 29 PM](https://user-images.githubusercontent.com/1720093/79021662-22364600-7b4a-11ea-8dad-71fa49063b47.png)